.github/workflows/build.yml

```diff
@@ -9,9 +9,9 @@ jobs:
       - name: "checkout repository"
         uses: "actions/checkout@v2"
       - name: "setup java"
-        uses: actions/setup-java@v2
+        uses: "actions/setup-java@v2"
         with:
-          distribution: 'adopt'
-          java-version: '17'
+          distribution: "adopt"
```

Comment:
Should be `temurin` (or another distribution) as adopt is no longer updated.

Reply:
https://github.com/PaperMC/bibliothek/pull/76/commits/b286b1e95fefacb1b817312bb982099feac98011